Aircompany/Aircompany/Airport.cs

```diff
@@ -1,94 +1,63 @@
 ﻿using Aircompany.Models;
 using Aircompany.Planes;
-using System;
 using System.Collections.Generic;
 using System.Linq;
 
 namespace Aircompany
 {
     public class Airport
     {
-        public List<Plane> Planes;
-
-        public Airport(IEnumerable<Plane> planes)
-        {
-            Planes = planes.ToList();
-        }
+        public List<Plane> Planes { get; set; }
 
         public List<PassengerPlane> GetPassengersPlanes()
         {
-            List<PassengerPlane> passengerPlanes = new List<PassengerPlane>();
-            for (int i=0; i < Planes.Count; i++)
-            {
-                if (Planes[i].GetType() == typeof(PassengerPlane))
-                {
-                    passengerPlanes.Add((PassengerPlane)Planes[i]);
-                }
-            }
-            return passengerPlanes;
+            return Planes.Where(plane => plane.GetType() == typeof(PassengerPlane)).Select(plane => plane as PassengerPlane).ToList();
         }
 
         public List<MilitaryPlane> GetMilitaryPlanes()
         {
-            List<MilitaryPlane> militaryPlanes = new List<MilitaryPlane>();
-            for (int i = 0; i < Planes.Count; i++)
-            {
-                if (Planes[i].GetType() == typeof(MilitaryPlane))
-                {
-                    militaryPlanes.Add((MilitaryPlane)Planes[i]);
-                }
-            }
-            return militaryPlanes;
+            return Planes.Where(plane => plane is MilitaryPlane).Select(plane => plane as MilitaryPlane).ToList();
         }
 
         public PassengerPlane GetPassengerPlaneWithMaxPassengersCapacity()
         {
             List<PassengerPlane> passengerPlanes = GetPassengersPlanes();
-            return passengerPlanes.Aggregate((w, x) => w.PassengersCapacityIs() > x.PassengersCapacityIs() ? w : x);             
+            var maxPassengerCapacity = passengerPlanes.Max(plane => plane.PassengersCapacity);
+            return passengerPlanes.First(x => x.PassengersCapacity == maxPassengerCapacity);
         }
 
         public List<MilitaryPlane> GetTransportMilitaryPlanes()
         {
-            List<MilitaryPlane> transportMilitaryPlanes = new List<MilitaryPlane>();
-            List<MilitaryPlane> militaryPlanes = GetMilitaryPlanes();
-            for (int i = 0; i < militaryPlanes.Count; i++)
-            {
-                MilitaryPlane plane = militaryPlanes[i];
-                if (plane.PlaneTypeIs() == MilitaryType.TRANSPORT)
-                {
-                    transportMilitaryPlanes.Add(plane);
-                }
-            }
-
-            return transportMilitaryPlanes;
+            return Planes.Where(plane => plane is MilitaryPlane && ((MilitaryPlane)plane).Type == MilitaryType.Transport).Select(plane => plane as MilitaryPlane).ToList();
```

Comment:
You used GetPassengersPlanes in previous method, why not to use GetMilitaryPlanes here?